src/views/3-Modeling/Modeling.vue

```diff
@@ -621,51 +626,50 @@
             id="legend-scale"
             transform="translate(0, 700)"
           >
-            <path
-              d="M228,643.5H807"
-              transform="translate(-28.24 -627.46)"
-              style="fill: none;stroke: white;stroke-miterlimit: 10;stroke-width: 3px"
-            />
+
             <text
-              class="text-annotate"
-              transform="translate(50 27.42)"
+              class="text-annotate left"
+              transform="translate(50 27)"
             >accurate</text>
             <text
-              class="text-annotate"
-              transform="translate(795 27.42)"
+              class="text-annotate right"
+              transform="translate(850 27)"
             >inaccurate</text>
-            <path
-              d="M247.5,656.5l-23-12,22-15"
-              transform="translate(-28.24 -627.46)"
-              style="fill: none;stroke: white;stroke-linecap: round;stroke-linejoin: round;stroke-width: 3px"
-            />
-            <path
-              d="M722.5,657.5l24-14-25-13"
-              transform="translate(30 -627.46)"
-              style="fill: none;stroke: white;stroke-linecap: round;stroke-linejoin: round;stroke-width: 3px"
-            />
+ 
           </g>
         </svg>
       </div>
     </figure>
     <!--     all the scrolling elements are created from modelingText.js content -->
     <article>
-      <div
-        v-for="text in text.methods"
-        :key="text.title"
-        class="step-container text-content"
-      >
+      <div id="scrollama-container">
+      <!-- create scrolling/sticky headers for each model section -->
```

Comment:
Nice! Good call here, I think having the section headings remain in place helps keep the viewer oriented to what model is being discussed